tests/integration-tests/configs/develop.yaml

```diff
@@ -77,7 +77,7 @@ test-suites:
       dimensions:
         - regions: ["eu-central-1"]
           instances: {{ common.INSTANCES_DEFAULT_X86 }}
-          oss: ["ubuntu1804"]
+          oss: ["ubuntu1804"]  # os must be different from centos7 to test os validation logic when wrong os is provided
```

Comment:
don't forget to add the scheduler now that the test function uses it as fixture. same below.
was it working without having the scheduler specified here?

Reply:
Done. I am not sure, because I was using test_runner to run tests locally